### PR TITLE
[KYUUBI #1660] Add restart option for bin/kyuubi

### LINKED ISSUE #1660: [FEATURE] Add `restart` option for `bin/kyuubi`
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the feature

if the server is running, perform stop and start, otherwise, start

### Motivation

_No response_

### Describe the solution

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Narcasserun**
- Add restart option for bin/kyuubi

This closes #1660

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]

File: bin/kyuubi [hunk 1]
```diff
@@ -23,6 +23,7 @@ function usage() {
   echo "Usage: bin/kyuubi command"
   echo "  commands:"
   echo "    start        - Run a Kyuubi server as a daemon"
+  echo "    restart      - restart Kyuubi server as a daemon"
```
Comment (turboFei) [inline bin/kyuubi:26]: `- restart` -> Restart

File: bin/kyuubi [hunk 2]
```diff
@@ -202,6 +203,13 @@ case $1 in
     start_kyuubi
     ;;
 
+  (restart)
+    echo "kyuubi is Restarting.......... "
```
Comment (turboFei) [inline bin/kyuubi:207]: maybe
```
Restarting Kyuubi
```

and 
```
Starting Kyuubi
```
?

Comment (Narcasserun) [reply bin/kyuubi]: Thanks.

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**